front/components/sparkle/AppLayout.tsx

```diff
@@ -419,14 +435,21 @@ export default function AppLayout({
           >
             <div
               className={classNames(
-                "mx-auto h-full",
-                isWideMode ? "w-full" : "max-w-4xl px-6"
+                "mx-auto h-full w-full",
+                isWideMode ? "w-full" : "max-w-4xl"
               )}
```

Comment:
@Duncid Having trouble here do you have a lead? My current status is that 
I'm in a situation where fixing one breaks the other one.
<img width="512" alt="Screenshot 2024-06-05 at 15 20 05" src="https://github.com/dust-tt/dust/assets/32683010/14445e39-85f5-4e0c-8d7a-94fa73abae96">
<img width="520" alt="Screenshot 2024-06-05 at 15 19 00" src="https://github.com/dust-tt/dust/assets/32683010/afdd8988-e15b-41d8-9c1b-a8d863022af6">